pkg/stub/handler.go

```diff
@@ -527,6 +534,13 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 			return nil
 		}
 
+		// Every time we see a change to the SamplesResource object, update the ClusterOperator status
+		// based on the current conditions of the SamplesResource.
+		err := operatorstatus.UpdateOperatorStatus(srcfg)
+		if err != nil {
+			return err
```

Comment:
We should have a new Condition on the SamplesResource status that says whether we updated the CVO status or not, no?

Reply:
i don't think that's a thing an end-user needs to care about.  for now i'm not inclined to add it.

Reply:
ok